public/stylesheets/main.css

```diff
@@ -37,3 +38,77 @@ h3 {
 }
 
 /* TODO - Add your own styles here */
+
+/* Headings */
+
+#page-title {
+font-style: italic;
```

Comment:
Try and get in the habit of indenting these to make it easier for others to read.